test/server.js

```diff
@@ -52,6 +50,3 @@ const portHTTPS = 3002;
 
 serverHTTP.listen(portHTTP);
 serverHTTPS.listen(portHTTPS);
-
-console.log("HTTP listening on " + portHTTP);
-console.log("HTTPS listening on " + portHTTPS);
```

Comment:
same as above, these are 2 `console.log` statements that would be nice to keep as well